deploy/run_with_summon.sh

```diff
@@ -5,39 +5,44 @@ set -xeuo pipefail
 
 # Clean up when script completes and fails
 function finish {
-  announce 'Wrapping up and removing test environment'
-
   # There is a TRAP in test_in_docker.sh to account for Docker deployments so we do not need to add another one here
   # Stop the running processes
-  if [[ $RUN_IN_DOCKER = false ]]; then
+   if [[ $RUN_IN_DOCKER = false && $DEV = false ]]; then
+    announce 'Wrapping up and removing dev environment'
```

Comment:
if dev is false then how come we are removing the dev env?

Reply:
because this will teardown the dev environment after it runs to completion which is not what we want. we want the dev env to stay up

Reply:
i see that. however, the text says `removing dev environment` which is incorrect. it removes the test env.

Reply:
Updating to `Wrapping up and removing environment`